### PR TITLE
Set up and venue sign up

### PR DESCRIPTION
**Pull request by Danwhy**
- Sets up new Phoenix project
- Adds initial signup flow for venues
- Displays initial basic venue page with details

Related issues:
#53 #54 #25

### CODE REVIEW

File: assets/brunch-config.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
+exports.config = {
```
Comment (nelsonic) [inline assets/brunch-config.js]: do we _need_ brunch in this project?
(_are we doing any JS or CSS compilation_?)
Or is it just for page reloading?

File: assets/package.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  "scripts": {
+    "deploy": "brunch build --production",
+    "watch": "brunch watch --stdin"
+  },
+  "dependencies": {
+    "phoenix": "file:../deps/phoenix",
+    "phoenix_html": "file:../deps/phoenix_html"
+  },
+  "devDependencies": {
+    "babel-brunch": "6.1.1",
```
Comment (nelsonic) [inline assets/package.json]: `babel-brunch` ... 🤔 (_as above/before, do we **need* brunch?_)

File: lib/cs_guide/accounts/accounts.ex [hunk 3]
```diff
@@ -0,0 +1,104 @@
+defmodule CsGuide.Accounts do
```
Comment (nelsonic) [inline lib/cs_guide/accounts/accounts.ex]: is this _generic_ Account? or is it required for registering a venue?

**Comment by Danwhy** [comment]
@nelsonic The Append Only functionality hasn't yet been implemented, but I agree it would be good to get it in ASAP before we do the initial deployment, so I'll add that in this morning

File: assets/brunch-config.js (hunk 1 above)
Comment (Danwhy) [reply assets/brunch-config.js]: We don't necessarily need brunch right now, but we are using it to build our static files by default. We could take some time to remove it if necessary

File: lib/cs_guide/accounts/accounts.ex (hunk 3 above)
Comment (Danwhy) [reply lib/cs_guide/accounts/accounts.ex]: This is just the context for the different account functions we might have. We're not currently using it, but it's created as part of the Phoenix generator, and will have some content in the future

**Comment by Danwhy** [comment]
@nelsonic The Append only log has now been fully implemented in this project, including updates and deletions

**Comment by nelsonic** [comment]
please see: https://github.com/dwyl/alog/issues/1

**Comment by Danwhy** [comment]
@nelsonic :+1: I'm confident it should be fairly quick to extract the append only module as it's very self contained. 
Did you want me to do that before this PR gets merged, or is it OK to deploy this then work on replacing the append only module?

File: assets/brunch-config.js (hunk 1 above)
Comment (Danwhy) [reply assets/brunch-config.js]: Now that we're using Elm, brunch is a bit more of a necessity to compile it with our javascript. We could still replace it with something else, but unless there's a clear advantage, I'm happy to stick with it

**Comment by nelsonic** [comment]
@danwhy please deploy the *branch* to Heroku as a “staging” version so CS can start testing it.
**.then** work on splitting out the append-only log into a re-useable module ASAP so we get it in and don’t have as much refactoring to do.
Thanks! 🙌

**Comment by nelsonic** [comment]
PR pending integration of https://github.com/dwyl/alog 👍

File: mix.exs [hunk 4]
```diff
@@ -41,7 +41,8 @@ defmodule CsGuide.Mixfile do
       {:phoenix_live_reload, "~> 1.0", only: :dev},
       {:gettext, "~> 0.11"},
       {:cowboy, "~> 1.0"},
-      {:autoform, git: "https://github.com/dwyl/autoform.git", tag: "0.1"}
+      {:autoform, git: "https://github.com/dwyl/autoform.git", tag: "0.1"},
+      {:alog, path: "/Users/Daniel/dwyl/alog"}
```
Comment (nelsonic) [inline mix.exs]: this will probably only work on your `localhost` ... 😉

File: assets/brunch-config.js (hunk 1 above)
Comment (nelsonic) [reply assets/brunch-config.js]: Happy to use Brunch for now. 👍 
Phoenix 1.4 uses Webpack: https://github.com/phoenixframework/phoenix/blob/0bdb5b001915e437c2d2cf1b77a1c532ce45cde2/CHANGELOG.md#phxnew-installer
🙄

File: assets/package.json (hunk 2 above)
Comment (nelsonic) [reply assets/package.json]: we can remove this if it becomes untenable.

File: lib/cs_guide/accounts/accounts.ex (hunk 3 above)
Comment (nelsonic) [reply lib/cs_guide/accounts/accounts.ex]: @Danwhy thanks for clarifying. we can update as required laters. good for now. 👍

File: lib/cs_guide_web/templates/user/new.html.eex [hunk 5]
```diff
@@ -0,0 +1,50 @@
+<section class="center w-80 w-75-l mv4">
+  <h3 class="f4 lh4">Create a Profile</h3>
+  <p class="mt3">You need to be part of the Club Soda community to add listings. All it
+  takes is your email and a password.</p>
+  <%= render_autoform(@conn, :create, CsGuide.Accounts.User, assigns: [changeset: @user_changeset], exclude: [:entry_id, :deleted]) %>
```
Comment (nelsonic) [inline lib/cs_guide_web/templates/user/new.html.eex]: consider renaming the `render_autoform` to `autoform_render` 🤔

File: mix.exs [hunk 6]
```diff
@@ -0,0 +1,62 @@
+defmodule CsGuide.Mixfile do
+  use Mix.Project
+
+  def project do
+    [
+      app: :cs_guide,
+      version: "0.0.1",
+      elixir: "~> 1.4",
```
Comment (nelsonic) [inline mix.exs]: does not match version we're testing with in `travis.yml` consider updating in next iteration.

**Pull request merged**